optuna/samplers/_cmaes.py

```diff
@@ -560,7 +560,12 @@ def _init_optimizer(
             for i, dist in enumerate(trans._search_space.values()):
                 assert isinstance(dist, (IntDistribution, FloatDistribution))
                 # Set step 0.0 for continuous search space.
-                steps[i] = dist.step or 0.0
+                if dist.step is None:
+                    steps[i] = 0.0
+                elif dist.low == dist.high:
+                    steps[i] = 1.0
+                else:
+                    steps[i] = dist.step / (dist.high - dist.low)
```

Comment:
As we discussed privately, this ad-hoc transform of `step` might easily cause a bug (e.g., by changing `_SearchSpaceTransform` options). How about adding `step` property to `_SearchSpaceTransform`?

Reply:
Memo: This PR is a hotfix and we will work on this as a follow-up work.